lib/install/management/finder.go

```diff
@@ -60,10 +59,10 @@ func (d *Dispatcher) NewVCHFromID(id string) (*vm.VirtualMachine, error) {
 		d.op.Errorf("Failed to find VM %q: %s", moref, err)
 		return nil, err
 	}
-	vmm = vm.NewVirtualMachine(d.op, d.session, ovm.Reference())
+	d.appliance = vm.NewVirtualMachine(d.op, d.session, ovm.Reference())
 
 	// check if it's VCH
-	if ok, err = d.isVCH(vmm); err != nil {
+	if ok, err = d.isVCH(d.appliance); err != nil {
```

Comment:
doubt it matters, but good practice would be to only assign to `d.appliance` after we've confirmed that it's a VCH.

Reply:
agree..but agree in this case it wouldn't matter...